### PR TITLE
Add "Edit" buttons to enable edit site log items

### PR DESCRIPTION
**Pull request by frankfu03**

### CODE REVIEW

File: src/client/app/shared/abstract-groups-items/abstract-base.component.ts [hunk 1]
```diff
@@ -5,9 +5,12 @@ import { SiteLogService } from '../site-log/site-log.service';
 export abstract class AbstractBaseComponent implements OnDestroy {
 
     public isEditable: boolean;
+    public isItemEditable: boolean;
```
Comment (lbodor) [inline src/client/app/shared/abstract-groups-items/abstract-base.component.ts]: Attributes isEditable and isItemEditable are too similar.

File: src/e2e/specs/page-objects/log-item-group.pageobject.ts [hunk 2]
```diff
@@ -56,6 +56,11 @@ export class LogItemGroup {
         return itemContainer.element(by.buttonText('Delete'));;
     }
 
+    public getEditButton(): ElementFinder {
+        let itemContainer: ElementFinder = this.getItemContainer(this.newItemIndex);
+        return itemContainer.element(by.buttonText('Edit'));;
```
Comment (lbodor) [inline src/e2e/specs/page-objects/log-item-group.pageobject.ts]: too many semicolons

File: src/client/app/shared/abstract-groups-items/abstract-base.component.ts (hunk 1 above)
Comment (frankfu03) [reply src/client/app/shared/abstract-groups-items/abstract-base.component.ts]: Yes, I do it on purpose as isEditable should be isAuthorised in fact. I may do it in a separate PR.

File: src/e2e/specs/page-objects/log-item-group.pageobject.ts (hunk 2 above)
Comment (frankfu03) [reply src/e2e/specs/page-objects/log-item-group.pageobject.ts]: yeah, extra semicolon is removed.

**Comment by frankfu03** [comment]
If "revert" is to discard the changes and changes the fields back to read-only, wouldn't "cancel" be more appropriate?

**Comment by lbodor** [comment]
I agree, cancel would be better.

**Comment by frankfu03** [comment]
This PR has too much changes and rollbacks. I would like to close it and open another clean PR.

**Pull request closed without merging**